python_modules/dagster/dagster/_core/test_utils.py

```diff
@@ -718,3 +724,21 @@ def ensure_dagster_tests_import() -> None:
         dagster_package_root / "dagster_tests"
     ).exists(), "Could not find dagster_tests where expected"
     sys.path.append(dagster_package_root.as_posix())
+
+
+def resolve_asset_job(
+    assets: Sequence[Union[AssetsDefinition, SourceAsset]],
+    *,
+    selection: Optional[CoercibleToAssetSelection] = None,
+    name: str = "asset_job",
+    resources: Mapping[str, object] = {},
+    **kwargs: Any,
+) -> JobDefinition:
+    assets_defs = [a for a in assets if isinstance(a, AssetsDefinition)]
+    source_assets = [a for a in assets if isinstance(a, SourceAsset)]
+    selection = selection or assets_defs
+    return Definitions(
+        assets=[*assets_defs, *source_assets],
+        jobs=[define_asset_job(name, selection, **kwargs)],
+        resources=resources,
+    ).get_job_def(name)
```

Comment:
why call this "resolve" asset job? 

`create_asset_job_from_assets`? or something?

Reply:
Good point, it was bc we are getting the result of `define_asset_job(...).resolve(...)` but "resolution" here is kind of an implementation detail. Changed to `create_test_asset_job`.